pkg/env/dev.go

```diff
@@ -74,7 +74,12 @@ func (d *dev) AROOperatorImage() string {
 		return override
 	}
 
-	return fmt.Sprintf("%s/aro:%s", d.ACRDomain(), version.GitCommit)
+	operatorAcr := os.Getenv("ARO_OPERATOR_ACR")
+	if operatorAcr == "" {
+		operatorAcr = d.ACRDomain()
+	}
+
+	return fmt.Sprintf("%s/aro:%s", operatorAcr, version.GitCommit)
```

Comment:
I don't see `ARO_OPERATOR_ACR` mentioned elsewhere in the code base. Is this additional env var to help format the image name?

Why wouldn't we put this format into the value of `ARO_IMAGE`?

Reply:
Added a comment in the docs. I thought the option was a good idea, considering the iterative changes I was making on the Operator.

Reply:
ARO_IMAGE is a static image -- e.g. if you want to use the prebuilt one. ARO_OPERATOR_ACR sets a custom Docker repo as the base, that then has the hash put on it.

We don't want to set the global ACR_RESOURCE_ID to it because then things like fluentbit, MDSD, etc, will look at that. We want to use the int ACR for all of those resources, but our custom ACR specifically for the Operator.